vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -104,3 +117,22 @@ struct AutoContributeProps {
   bool contribution_videos;
   uint64 reconcile_stamp;
 };
+
+struct TransactionStatementInfo {
+  string probi;
+  uint64 date;
+  int32 category;
+};
+
+struct MonthlyStatements {
```

Comment:
I think we should avoid having plural types. We can have a MonthlyStatement type and pass an array of them, but that array does not require a new type.